src/sdkV2.ts

```diff
@@ -61,6 +61,10 @@ export default class CegaEvmSDKV2 {
     return chainConfig;
   }
 
+  async getProvider(): Promise<ethers.providers.Provider> {
```

Comment:
I need the provider to load `ERC20` contract so I can fetch name, decimals, and symbol. With the way we're loading the sdk for the subgraph stub requests, it would be easier to just fetch the provider from the sdk

Can make this not async

Reply:
refactored and got rid of this